docs/concepts/workers.md

```diff
@@ -52,4 +52,4 @@ It is solely responsible for orchestrating state transitions and dispatching mes
 
 While data transferred in the event histories is [secured by mTLS](https://docs.temporal.io/docs/server/security/#encryption-of-network-traffic), by default, it is still readable at rest in the Temporal Server.
 
-To solve this, Temporal SDKs offer a [Data Converter API](https://docs.temporal.io/docs/java/activities/#activity-interface) that you can use to customize the serialization of data going out of and coming back in to a Worker, with the net effect of guaranteeing that the Temporal Server cannot read sensitive business data.
+To solve this, Temporal SDKs offer a [Data Converter API](/docs/content/what-is-a-data-converter) that you can use to customize the serialization of data going out of and coming back in to a Worker, with the net effect of guaranteeing that the Temporal Server cannot read sensitive business data.
```

Comment:
Instead of "To solve this," perhaps use "To address this vulnerability," assuming of course that my change doesn't overstate the issue.